main.rb

```diff
@@ -1,12 +1,61 @@
-require_relative('./person')
-require_relative('./trimmer')
-require_relative('./capitalize')
-require_relative('./nameable')
-require_relative('./decorator')
-p person = Person.new(22, 'maximilianus')
-p person.correct_name
-# capitalizedPerson = Capitalize.new(person)
-# p capitalizedPerson
-# p capitalizedPerson.correct_name
-# capitalizedTrimmedPerson = Trimmer.new(capitalizedPerson)
-# p capitalizedTrimmedPerson.correct_name
+require './app'
+
+class Main
```

Comment:
when I tried to run the main.rb file I got an error , Kindly fix this issue 👍 👍 

here is the error I got : 
![image](https://user-images.githubusercontent.com/87448628/181906281-9fb82a87-5892-415a-8947-392aca9e9279.png)

Try to check the arguments in the person class initialize method 👍